fastly/ftp.go

```diff
@@ -15,11 +15,12 @@ type FTP struct {
 	Name              string     `mapstructure:"name"`
 	Address           string     `mapstructure:"address"`
 	Port              uint       `mapstructure:"port"`
-	Username          string     `mapstructure:"user"`
+	User              string     `mapstructure:"user"`
```

Comment:
Unfortunately, we can't make this change because it would break backward compatibility.